accelerator/roles/intel/tasks/install_ubuntu.yml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Check if accelerator is present on node
+  ansible.builtin.include_tasks: verify_has_accelerators.yml
+
+- name: Include accelerator_config
+  ansible.builtin.include_vars: "{{ role_path }}/../../../input/accelerator_config.yml"
+
+- name: Install drivers on Gaudi nodes
+  when: node_has_accelerator
+  block:
+    - name: Gather package facts
+      ansible.builtin.package_facts:
+        manager: auto
+
+    - name: Check if kernel supported
+      ansible.builtin.fail:
+        msg: "Kernel not supported"
+      when: ansible_kernel is version('5.4.0', '<')
+
+    - name: Base dependencies
+      ansible.builtin.apt:
+        name: "{{ intel_apt_base_packages | list }}"
+        state: latest
+        update_cache: true
+
+    # This number was defined by Habana team and is the number used in IDC clusters
+    - name: Set the number of hugepages
```

Comment:
installing tuned packages will be part of different utility and omnia team is writing playbook for the same

Reply:
So, just to be clear, setting the number of hugepages here is required by Gaudi for running certain workloads. This value is not being set by any default tuned profile. I don't think this should be removed from the Gaudi provisioning task.

Reply:
we will be creating seperate utility to set such specific parameters for any GPU. And that will be called in Omnia.yml, hence we can remove from here

Reply:
ok we will remove this